src/Hangman/HangmanGame.java

```diff
@@ -0,0 +1,208 @@
+            }
+            updateBoard(board, lettersToGuess, guess, word);
+        }
+        printBoard();
+        printLetters();
+
+        input.nextLine();
+        System.out.println("Would you like to play again? (Y/N)");
+        String choice = input.nextLine();
+        while(!choice.toLowerCase().equals("y") & !choice.toLowerCase().equals("n")){
+            System.out.println("Please only select Y or N");
+            choice = input.next();
+        }
+        if(choice.toLowerCase().equals("y")){
+            resetStats();
+            startGame();
+
+        }
+        else printEndBoard();
+
+    }
+
+    public void messageToPlayer(){
+        System.out.println("Letters Guessed: " + lettersGuessed);
+        if(incorrectGuesses == 6) System.out.println("Sorry you LOSE! The correct word was: " + word);
+        else if(correctGuesses == lengthOfWord) System.out.println("Congratulations you WON!");
+        else System.out.println("Please Guess a Letter: ");
+    }
+
+    public void updateBoard(String[] board, String[] lettersToGuess, String guess, String word){
+        if(!word.contains(guess) & !lettersGuessed.contains(guess)){
+            lettersGuessed.add(guess);
+            incorrectGuesses++;
+            switch (String.valueOf(incorrectGuesses)) {
+                case "1":
+                    board[1] = "  O  │";
+                    break;
+                case "2":
+                    board[2] = " /   │";
+                    break;
+                case "3":
+                    board[2] = " /|  │";
+                    break;
+                case "4":
+                    board[2] = " /|\\ │";
+                    break;
+                case "5":
+                    board[3] = " /   │";
+                    break;
+                case "6":
+                    board[3] = " / \\ │";
+                    break;
+                default:
+                    System.out.println("Game Over");
+            }
+        }
+        else {
+            if(!lettersGuessed.contains(guess)){
+            for (int i = 0; i < word.length(); i++) {
+                if(guess.equals("" + word.charAt(i))){
+                        lettersToGuess[i] = " " + guess + " ";
+                        lettersGuessed.add(guess);
+                        correctGuesses++;
+                }
+            }
+        }
+        }
+
+    }
+
+    public void resetStats(){
+        incorrectGuesses = 0;
+        correctGuesses = 0;
+        lettersGuessed.clear();
+    }
+
+    public static Boolean verifySingleLetterGuess(String guess){
+        return guess.length() == 1 & Character.isLetter(guess.charAt(0));
+    }
+
+    public String getRandomWord(String[] words) {
+        Random random = new Random();
+        int index = random.nextInt(words.length);
+        return words[index];
+    }
+
+    private static String[] wordList = {"apple", "banana", "orange", "grape", "pineapple", "watermelon", "kiwi", "strawberry",
```

Comment:
This would be a fun implementation detail just to get your hands wet with apis and json. Instead of having this array of strings, you could have a class that fetches an api that returns a list of words. Then use that to generate words for the game. 

Something fun to do!